sys/net/lora/Makefile

```diff
@@ -0,0 +1 @@
+include $(RIOTBASE)/Makefile.base
```

Comment:
I don't think this is needed...

Reply:
I found the same in empty repositories like [these](https://github.com/RIOT-OS/RIOT/tree/master/sys/net/routing) hence added the same.

Reply:
It seems to be forgotten from when NHDP was removed in #13396. It's not needed.